build.sbt

```diff
@@ -15,33 +14,34 @@ def scalaBinaryVersion(scalaVersion: String) = scalaVersion match {
   case version if version startsWith "2.12" => "2.12"
   case _ => sys.error("unknown error")
 }
-def isScala210x(scalaVersion: String) = scalaBinaryVersion(scalaVersion) == "2.10"
 
 val algebirdVersion = "0.13.0"
 val apacheCommonsVersion = "2.2"
 val avroVersion = "1.7.4"
-val bijectionVersion = "0.9.1"
+val bijectionVersion = "0.9.5"
 val cascadingAvroVersion = "2.1.2"
-val chillVersion = "0.8.3"
+val chillVersion = "0.8.4"
```

Comment:
@johnynek summingbird with storm 1.0 support wasn't released because I'm waiting on scalding/storehaus to be released with 0.8.x chill.

In order to release SB with 0.9 storm and 2.12 scala support we also need to release tormenta with the same. Latest tormenta's release uses storm 1.0 (which is not good for above) and supports 2.12.

So I see two possibilities:
1. If we really want to release SB with 0.9 storm and 2.12 scala:
- release old version of tormenta with 2.12 support only, without 1.0 storm
- release scalding 0.17.0 as @piyushnarang said
- release storehaus & SB with 2.12 & 0.9 storm.

And iterate again with scalding/storehaus/SB releases to get SB with 2.12 & 1.0 storm.

2. Or if we are ok without SB for 2.12 and 0.9 storm:
- release scalding with 0.8.x chill & 2.12 support
- release storehaus/SB with 2.12 & 1.0 storm

One more note - we use SB with 1.0 storm at Twitter for last couple of months and it works well. Transition to new api wasn't painful.